src/content/blog/capstone_UW_24.md

```diff
@@ -0,0 +1,56 @@
+Quantum error mitigation (QEM) is an active field seeking to reduce the effects of noise when performing computation on contemporary quantum devices. Several QEM techniques have been developed that vary in performance and overhead, many of which Mitiq offers implementations for. However, individuals seeking to perform mitigation may have different priorities regarding the consumption of resources and performance improvement, and Mitiq does not currently address how its implementations vary in this regard. This motivates the need for a tool that can characterize and compare the overhead of these techniques, providing insight into which method may best suit one’s interests. 
+
+### General Timeline:
+Week 1-2: Diving into Mitiq and Quantum Computing We started by familiarizing ourselves with Mitiq’s architecture and functionalities. This involved reading documentation, playing around with some of the error mitigation tutorials, and asking many questions.
+
+Week 3-5: Executors, MetaData, and the GUI These weeks we became accustomed with Mitiq’s executor functions, establishing our analysis pipeline by first focusing on Mitiq’s Zero-Noise Extrapolation (ZNE) implementation. We also set up metadata generation, and started GUI integration.
+
+Week 6-8: Testing and Refining With the core components in place, we spent this time extending our analysis to work for other Quantum Error Mitigation methods:  Probabilistic Error Cancellation (PEC),  Digital Dynamical Decoupling (DDD), and Readout Error Mitigation (REM).
+
+Week 9-10: Finalizing and Presentation In the final weeks, we added finishing touches to the pipeline and GUI and prepared for their presentations. We participated in Unitary Fund’s Quantum Wednesday and presented at a couple of our university’s poster sessions.
+
+### Results
+In order to directly compare certain overhead parameters for each QEM technique, we ran a standardized experiment using each one. We generated 10 mirror circuits— a type of quantum benchmarking circuit— each with fixed depth and number of shots, increasing the numbers of qubits. We used two homemade noise models for running on simulators - a thermal relaxation model with an added readout error, and a depolarizing noise model with single and two-qubit errors. Prior to running any sweeps involving mitigation, we ensured that our pre-mitigated expectation values were comparable using both noise models, thus ensuring our initial results were standardized.We then  ran our sweeps over the circuit qubit number with error mitigation and  recorded  various measures of overhead required when using the mitigation. These included the additional number of single and two-qubit gates required, the total added circuit depth, the number of additional unique circuits needed to be run, and  the additional time required to run each technique. The "no error mitigation" comparison was run on our thermal noise model. The results for the time requirements are are shown in the graphs below:

```

Comment:
```suggestion
In order to directly compare certain overhead parameters for each QEM technique, we ran a standardized experiment using each technique. We generated 10 mirror circuits---a type of quantum benchmarking circuit---each with fixed depth and number of shots, increasing the numbers of qubits. We used two tailored noise models run on simulators---a thermal relaxation model with an added readout error, and a depolarizing noise model with single and two-qubit errors. Prior to running any sweeps involving mitigation, we ensured that our pre-mitigated expectation values were comparable using both noise models, thus ensuring our initial results were standardized. We then ran sweeps over the circuit qubit number with error mitigation and recorded various measures of overhead required when using the mitigation. Overhead measurements included the additional number of single and two-qubit gates required, the total added circuit depth, the number of additional unique circuits needed to be run, and the additional time required to run each technique. The "no error mitigation" comparison was run on our thermal noise model. The results for the time requirements are are shown in the graphs below.
```

I've made some suggested edits here. They don't seem to be highlighted like above since the line is so long. For this reason some people put a line limit on the number of characters per line, or even one sentence per line. Y'all don't have to do that here, just FYI!